website/docs/blocks.md

```diff
@@ -62,15 +62,15 @@ Response:
 
 ## Abort blocks
 
-This functionality allows simulating block abortion that can occur on mainnet.
+This functionality allows simulating block abortion that can occur on mainnet, requires `-state-archive-capacity full` CLI option.
```

Comment:
A dash is missing, but the clarity could be improved, too:
```suggestion
This functionality allows simulating block abortion that can occur on mainnet. It is supported in the `--state-archive-capacity full` mode.
```

Reply:
34a300fdfc68193e7e92117ce93e7dde11f97e45